exporter/clickhouseexporter/config.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouseexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/clickhouseexporter"
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+	"go.uber.org/multierr"
+)
+
+// Config defines configuration for Elastic exporter.
+type Config struct {
+	config.ExporterSettings        `mapstructure:",squash"`
+	exporterhelper.TimeoutSettings `mapstructure:",squash"`
+	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
```

Comment:
`exporterhelper.RetrySettings` usually goes with `exporterhelper.QueueSettings` to control the queue size. Is there a reason to omit it?

Reply:
As ClickHouse recommend inserting data in packets of at least 1000 rows, or no more than a single request per second. 
The `exporterhelper.QueueSettings` has a field `NumConsumers`, So I omit the `exporterhelper.QueueSettings` to keep it only one exporter goroutine, it consume the data from batch-processor, which batch by size 1000 or time 1s.  

I rethink it again, may be use a subset field `QueueSettings` will be better, only keep `queue_size`,  like awslogexporter does. https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awscloudwatchlogsexporter/config.go#L80

Reply:
I have add `QueueSettings` field in latest commit (:

Reply:
Sounds good. thanks